src/HikConsole/HikClient.cs

```diff
@@ -1,181 +1,185 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using HikApi.Abstraction;
+using HikApi.Data;
 using HikConsole.Abstraction;
-using HikConsole.Data;
+using HikConsole.Config;
 using HikConsole.Helpers;
-using C = HikConsole.Helpers.ConsoleHelper;
 
 namespace HikConsole
 {
-    public class HikClient
+    public class HikClient : IHikClient
```

Comment:
It is better to make the Client state less and split state related stuff into another object